tokio-reactor/src/poll_evented.rs

```diff
@@ -85,7 +85,7 @@ use std::sync::atomic::Ordering::Relaxed;
 /// [`Registration`]: struct.Registration.html
 /// [`TcpListener`]: ../net/struct.TcpListener.html
 /// [`clear_read_ready`]: #method.clear_read_ready
-/// [`clear_read_ready`]: #method.clear_read_ready
+/// [`clear_write_ready`]: #method.clear_write_ready
```

Comment:
Maybe you should also change `std::Read` -> `std::io::Read` and `std::Read` -> `std::io::Read` (10 lines above) like it is done in `src/reactor/mod.rs`?

Reply:
Done, in fa53948 and a fine chance to appease the CI gods.  Just for the record, I neither authored the original `std::Read` style nor started the piecemeal conversation to the more accurate if verbose, `std::io::Read`. :-)

Reply:
Thank you